snewpdag/plugins/DiffPointing.py

```diff
@@ -48,25 +47,25 @@ def cache_values(self, k1, k2, dts):
     d1 = self.db.get(k1)
     d2 = self.db.get(k2)
     if 'dt' in dts and 't1' in dts and 't2' in dts:
-      dt = dts['dt'] # (s, ns)
-      t1 = dts['t1'] # (s, ns)
-      t2 = dts['t2'] # (s, ns)
+      dt = dts['dt'] # s
+      t1 = dts['t1'] # s
+      t2 = dts['t2'] # s
     else:
       return None
     bias = dts['bias'] if 'bias' in dts else d1.bias - d2.bias # sec
     var = dts['var'] if 'var' in dts else d1.sigma**2 + d2.sigma**2 # sec**2
     dsig1 = dts['dsig1'] if 'dsig1' in dts else d1.sigma # sec
     dsig2 = dts['dsig2'] if 'dsig2' in dts else - d2.sigma # sec
     # we actually want a time basis in ms for all times
-    g = 1000.0 / ns_per_second
+    g = 1000.0
```

Comment:
this is second to ms right? maybe use another name for g variable?

Reply:
legacy from Wiktor's original code, I think. Probably time to change it.